protostar/cli/command.py

```diff
@@ -32,6 +34,14 @@ def directory(arg: str) -> Path:
                 assert pth.is_dir(), f'"{str(pth)}" is not a valid directory path'
                 return pth
 
+            @staticmethod
+            def felt(arg: str) -> int:
+                try:
+                    [output] = cast_to_felts([arg])
+                except ValueError as ex:
+                    assert False, str(ex)
```

Comment:
@kasperski95 is there a need for raising `AssertionError` here? If not they I think @Radinyn we can remove this try..except clause

Reply:
By 👍🏻 reaction I understand we can remove try..except

Reply:
It's better to throw the exception.

We need to refine this code in the future by introducing `ArgumentValidationError` in order to tell the user which argument caused the error.